gnome-initial-setup/gis-driver.c

```diff
@@ -87,33 +87,55 @@ typedef struct _GisDriverPrivate GisDriverPrivate;
 
 G_DEFINE_TYPE_WITH_PRIVATE(GisDriver, gis_driver, GTK_TYPE_APPLICATION)
 
+/* Should be kept in sync with eos-installer */
 static gboolean
-running_live_session (void)
+check_for_live_boot (gchar **uuid)
```

Comment:
I'd prefer it if this parameter was optional; alternatively you could have two functions, one that detects live boots, and another one that detects the UUID. That way you could pass NULL (or avoid getting the UUID altogether) if desired.

Reply:
I have added `g_return_val_if_fail (uuid != NULL, FALSE);`.